test/gurka/test_gtfs.cc

```diff
@@ -575,7 +581,7 @@ TEST(GtfsExample, MakeProto) {
   std::vector<float> last_dest_dist_traveled;
 
   // get the full shape length
-  float shape_length = 0;
+  [[maybe_unused]] float shape_length = 0;
```

Comment:
or remove it entirely? or is it used in some LOG_TRACE or so?

Reply:
Variable is set later on,
```
shape_length += currOrigin.Distance(currDest);
```
but never used.
I will try to make a test out of it.

Reply:
sure if you want, feel free, that'd be ideal. I guess it was meant to be used, but wasn't implemented in the end.

but also, if it takes you too long, no problem just removing both lines.

Reply:
made a test